api/src/main/java/net/kyori/adventure/text/serializer/ComponentSerializer.java

```diff
@@ -55,9 +55,25 @@
    * @param input the input
    * @return the component if {@code input} is non-null, otherwise {@code null}
    * @since 4.7.0
+   * @deprecated Use the correctly spelt {@link #deserializeOrNull(Object)}.
```

Comment:
```suggestion
   * @deprecated for removal since 4.8.0, use {@link #deserializeOrNull(Object)} instead
```

Reply:
With `forRemoval = true` in the annotation as well?

Reply:
No, we are not on Java 9+.

Reply:
Added! Although it does feel weird having `@Contract` at the top :')